BackendBench/op_categories.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+TENSOR_CREATION_AND_MANIPULATION_OPS = [
+    "arange",
+    "as_strided",  # broken in Shahin's run
```

Comment:
let's not check in Shahin's comments in the code, instead put these in some excel spreadsheet we can keep auditing

Reply:
yeah planning to remove it before merge. This is for the reviewer to know "this is definitely something we want to skip"